Bumerang/app/src/main/java/com/seng480b/bumerang/fragments/CreateRequestFragment.java

```diff
@@ -75,6 +76,7 @@
     private ImageView chosenImage;
 
     CheckBox postToFacebook;
+    RadioButton borrow;
```

Comment:
We've already set up the radio button to automatically reflect the selected type in the requestType variable. I think you should change it to use the request type rather than the input from the radio button. Just my thoughts.